x-pack/plugins/observability_solution/logs_data_access/common/constants.ts

```diff
@@ -5,4 +5,4 @@
  * 2.0.
  */
 
-export const DEFAULT_LOG_SOURCES = ['logs-*-*'];
+export const DEFAULT_LOG_SOURCES = ['logs-*-*,logs-*,filebeat-*,kibana_sample_data_logs*'];
```

Comment:
What happens if we expand this later, for example add `logstash-*` or `auditbeat-*`? Will users upgrading automatically pick this up too? Same question for if we remove an entry?

Reply:
Just been testing this out - for clusters where the setting hasn't been adjusted new changes will be picked up.